liquibase-standard/src/main/java/liquibase/util/StringUtil.java

```diff
@@ -477,26 +448,10 @@ public static List<String> splitAndTrim(String s, String regex) {
         return returnList;
     }
 
-    /**
-     * @deprecated use {@link StringUtils#repeat(String, int)} instead
-     */
-    @Deprecated
-    public static String repeat(String string, int times) {
-        return StringUtils.repeat(string, times);
-    }
-
-    /**
-     * @deprecated use {@link StringUtils#join(Object[], String)} instead
-     */
-    @Deprecated
```

Comment:
Keep this one too if you can't remove the method below.

Reply:
Same here.